osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -187,13 +187,27 @@ private TriangleParticle createTriangle(bool randomY)
         {
             TriangleParticle particle = CreateTriangle();
 
-            particle.Position = new Vector2(nextRandom(), randomY ? nextRandom() : 1);
+            particle.Position = getRandomPosition(randomY, particle.Scale);
             particle.ColourShade = nextRandom();
             particle.Colour = CreateTriangleShade(particle.ColourShade);
 
             return particle;
         }
 
+        private Vector2 getRandomPosition(bool randomY, float scale)
+        {
+            float y = 1;
+
+            if (randomY)
+            {
+                // since triangles are drawn from the top - allow them to be positioned a bit above the screen
+                float maxOffset = triangle_size * scale * 0.866f / DrawHeight;
```

Comment:
Might wanna comment what `0.866f` is why and where it comes from (it's the height of an equilateral triangle, or `sqrt(3) / 2`).

Reply:
It's used in many places there, so I thought it's okay. Should I comment in all of them?

Reply:
I've moved it out for you 👍